docs/sources/review/lint-prose/index.md

```diff
@@ -59,3 +59,73 @@ The task title contains a verb and an object. For example: "Create a dashboard".
 
 <!-- vale Grafana.Quotes = YES -->
 ```
+
+## Use Vale in VS Code
+
+You can use Vale to lint your current document in VS Code.
+
+1. Clone the [Writer's Toolkit](https://github.com/grafana/writers-toolkit/) repository.
+
+   ```bash
+   git clone git@github.com:grafana/writers-toolkit.git
+   ```
+
+1. Download and install [Vale](https://github.com/errata-ai/vale/releases).
```

Comment:
```suggestion
1. Download and install [Vale](https://vale.sh/docs/vale-cli/installation/).
```

Maybe we can link to the Vale installation page here instead? If people are using `brew` or `choco`, they don't have to go the releases page.